.tav.yml

```diff
@@ -79,7 +79,11 @@ graphql:
   commands: node test/instrumentation/modules/graphql.js
 express:
   versions: '>=4'
-  commands: node test/instrumentation/modules/express/basic.js
+  commands:
+    - node test/instrumentation/modules/express/basic.js
+    - node test/instrumentation/modules/express/error-handdling-1.js
+    - node test/instrumentation/modules/express/error-handdling-2.js
+    - node test/instrumentation/modules/express/set-framework.js
```

Comment:
Forgot this when I made the set-framework tests previously. Will make a separate PR to update the other framework TAV tests